tests/dp_computations_test.py

```diff
@@ -837,6 +837,65 @@ def test_compute_mean_negative_dp_count(self, mock_sum_add_noise,
         mock_sum_add_noise.assert_called_with(21)
 
 
+class ThresholdingMechanismTests(parameterized.TestCase):
+
+    def create_thresholding_mechanism(
+        self,
+        is_gaussian: bool = True,
```

Comment:
nit: would it be cleaner and more explicit to pass aggregate_params.MechanismType as an input instead? 
Otherwise, the method secretly assumes that all mechanisms that aren't Gaussian are Laplace.